Makefile

```diff
@@ -169,6 +169,11 @@ artifacts: kustomize
 	$(KUSTOMIZE) build config/crd > ${RELEASE_PATH}/temporal-operator.crds.yaml
 	$(KUSTOMIZE) build config/default > ${RELEASE_PATH}/temporal-operator.yaml
 
+.PHONY: artifacts
+helm: manifests helmify
```

Comment:
As you've overwritten the generated chart, are the new Makefile targets still useful ? IMO we can remove them. 
Helmify was just a bootstraping tool, am I wrong ?

Reply:
Hum this target could still be revelent to push new CRDS to the helm chart when preparing release ?

Reply:
Seems the helmify is unnecessary in this case. Enough to copy `${RELEASE_PATH}/temporal-operator.crds.yaml` to the chart's crds directory.